controllers/azurejson_machine_controller.go

```diff
@@ -119,9 +118,9 @@ func (r *AzureJSONMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result,
 	}
 
 	// Fetch the Cluster.
-	cluster, err := util.GetOwnerCluster(ctx, r.Client, azureMachine.ObjectMeta)
+	cluster, err := util.GetClusterFromMetadata(ctx, r.Client, azureMachine.ObjectMeta)
 	if err != nil {
-		return reconcile.Result{}, err
+		return reconcile.Result{}, nil
```

Comment:
why not return the err here? that would cause it to not requeue

Reply:
yeah, should keep the err, (copy paste bug) 🤦‍♂️